iothub/device/samples/how to guides/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -149,14 +149,14 @@ private async Task InitializeAndSetupClientAsync(CancellationToken cancellationT
                     cancellationToken: cancellationToken);
                 _logger.LogDebug("The client has subscribed to cloud-to-device messages.");
 
-                await RetryOperationHelper.RetryTransientExceptionsAsync(
+                /*await RetryOperationHelper.RetryTransientExceptionsAsync(
```

Comment:
Plans here?

Reply:
thanks for catching this; I forgot to uncomment this